src/mqtt/MQTT.cpp

```diff
@@ -77,8 +77,7 @@ void MQTT::onReceive(char *topic, byte *payload, size_t length)
                         if (jsonPayloadStr.length() <= sizeof(p->decoded.payload.bytes)) {
                             memcpy(p->decoded.payload.bytes, jsonPayloadStr.c_str(), jsonPayloadStr.length());
                             p->decoded.payload.size = jsonPayloadStr.length();
-                            meshtastic_MeshPacket *packet = packetPool.allocCopy(*p);
-                            service.sendToMesh(packet, RX_SRC_LOCAL);
+                            service.sendToMesh(p, RX_SRC_LOCAL);
```

Comment:
is there a reason why you removed the alloc?

Reply:
Oh, yeah that's actually unrelated to this pull, but I just stumbled upon it. I believe this leads to a memory leak as `p` wouldn't be freed. I believe the copy is unnecessary here; the "sendtext" part also doesn't use it.

Reply:
We can try, sounds plausible ... (how does telemetry handle this? there was a leak in there too...)

Reply:
At least currently it only allocates a `MeshPacket` once and uses that in `sendToMesh()`, as it should be:
https://github.com/meshtastic/firmware/blob/d1a25947e302cff9c0f1cb9b636569b67960fa9f/src/modules/Telemetry/DeviceTelemetry.cpp#L91

Reply:
sounds about right, telemetry was one of the sources of memory leaks... one more down :-)